web-components/src/components/editable-textfield/scss/editable-textfield.scss

```diff
@@ -95,7 +95,8 @@
     }
 
     &--textoverflow {
-      display: box;
+      -webkit-box-orient: vertical;
```

Comment:
does this consistently wotk with FF, chrome, edge?

Reply:
this was the original code, broken due to code changes in PR:https://sqbu-github.cisco.com/Collaboration/momentum-ui-web-components/pull/329
Tested in Chrome, Firefox